### PR TITLE
Fix/import script

### PR DESCRIPTION
**Pull request by vivekjhauipath**

### CODE REVIEW

File: platform/onebox/backup_and_restore/registry/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Usage
+* Run get-credentials.sh on AIF machines. It generates a file registry-creds.json. Copy it over to the backup/restore VM.
+* [Optionally] Move the credentials to some cluster manager and make changes to the scripts to read from there
+# Registry Backup-Restore
+* Make sure to use absolute path as basepath in below scripts
+
+### For Backup
+```
+./backup.sh <path to creds file> <basePath to download assets/blobs>
```
Comment (SahilChaudhary25) [inline platform/onebox/backup_and_restore/registry/README.md:26]: lets change to import & export

File: platform/onebox/backup_and_restore/registry/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+```
+It creates a folder <basePath>/registry which contains 1 tar file for every image
+
+### For Restore
+```
+./restore.sh <path to creds file> <basePath to upload assets/blobs from>
```
Comment (SahilChaudhary25) [inline platform/onebox/backup_and_restore/registry/README.md:32]: same here

**Review by SahilChaudhary25 — COMMENTED** [review]
Minor docs comments rest LGTM

File: platform/onebox/backup_and_restore/registry/README.md (hunk 1 above)
Comment (vivekjhauipath) [reply platform/onebox/backup_and_restore/registry/README.md:26]: I will rename files

**Review by SahilChaudhary25 — APPROVED** [review]

**Review by SahilChaudhary25 — APPROVED** [review]

**Review by vineetsingh5 — APPROVED** [review]
👍

**Pull request merged**